gtsam/linear/tests/testNoiseModel.cpp

```diff
@@ -807,23 +807,22 @@ TEST(NoiseModel, NonDiagonalGaussian)
   }
 }
 
-TEST(NoiseModel, ComputeLogNormalizer) {
+TEST(NoiseModel, ComputeLogNormalizerConstant) {
```

Comment:
This is a good test for Isotropic, but we also need tests for Gaussian, Diagonal, and Unit :-/

Reply:
Added tests for all 4 noise models.